src/sentry/static/sentry/app/views/organizationIntegrations/integrationRepos.jsx

```diff
@@ -286,7 +286,9 @@ export default class IntegrationRepos extends AsyncComponent {
                       </Box>
                       <Box>
                         <small>
-                          <a href={repo.url}>{repo.url.replace('https://', '')}</a>
+                          <a href={repo.url}>
```

Comment:
This is probably a silly question, but what happens if this is null here?

Reply:
There won't be a visible link as react will convert `null` into `''`.

Reply:
Alternatively you could conditionally render the `<a>` e.g. `{repo.url && <a href={....}>}`